ci_build_images/hashicorp.Dockerfile

```diff
@@ -0,0 +1,17 @@
+
+# INSTALL HASHICORP VAULT
+# USAGE: vault server -dev > /dev/null 2>&1 &
+
+RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg \
+&& echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] \
+https://apt.releases.hashicorp.com $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/hashicorp.list \
```

Comment:
`lsb_release` is fragile and needs the command to be installed, use `/etc/os-release` instead. See other example in the repo.

Indentation could be improved:
```
RUN ...
  && ...
  &&
```